amazon-ec2-net-utils.spec

```diff
@@ -6,39 +6,26 @@
 
 Name:      amazon-ec2-net-utils
 Summary:   A set of network tools for managing ENIs
-Version:   1.4
-Release:   2%{?dist}
+Version:   1.5
+Release:   1%{?dist}
 License:   MIT and GPLv2
-Group:     System Tools
-Source0:   53-ec2-network-interfaces.rules.systemd
-Source1:   53-ec2-network-interfaces.rules.upstart
-Source2:   75-persistent-net-generator.rules
-Source3:   ec2net-functions
-Source4:   ec2net.hotplug
-Source5:   ec2ifup
-Source6:   ec2ifdown
-Source7:   ec2dhcp.sh
-Source8:   ec2ifup.8
-Source9:   ec2ifscan
-Source10:  ec2ifscan.8
-Source11:  ixgbevf.conf
-Source12:  elastic-network-interfaces.conf
-Source13:  ec2net-scan.service
-Source14:  write_net_rules
-Source15:  rule_generator.functions
-Source16:  ec2net-ifup@.service
+
+Source:    https://github.com/aws/amazon-ec2-net-utils/archive/%{version}.tar.gz
 
 URL:       https://github.com/aws/amazon-ec2-net-utils
 BuildArch: noarch
 Requires:  curl
 Requires:  iproute
+BuildRequires: make
+BuildRequires: systemd
 %if %{systemd}
 %{?systemd_requires}
 BuildRequires: systemd-units
 Requires: systemd-units
 %endif # systemd
 Requires: dhclient
-Provides: ec2-net-utils
+Provides: ec2-net-utils = %{version}-%{release}
+Obsoletes: ec2-net-utils < 1.5
```

Comment:
should this be `< %{version}-%{release}` ?

Reply:
I asked that same question in #13. Apparently it should not.